common-api/src/main/java/org/pmiops/workbench/config/WorkbenchConfig.java

```diff
@@ -45,13 +45,36 @@ public static WorkbenchConfig createEmptyConfig() {
     return config;
   }
 
+  // Environment config variables related to billing and the billing project buffer (which buffers
+  // GCP projects, aka "billing projects" in Terra terminology).
   public static class BillingConfig {
+    // This config variable seems to be unused.
     public Integer retryCount;
+    // The total capacity of the GCP project buffer. The buffering system will not attempt to create
+    // any new projects when the total number of in-progress & ready projects is at or above this
+    // level.
     public Integer bufferCapacity;
+    // The number of times to attempt project creation per cron task execution. This effectively
+    // controls the max rate of project refill. If the cron task is configured to run once per
+    // minute and this param is set to 5, then the buffer system will create up to 5 projects
```

Comment:
I looked up the GAE cron overrun policy here, looks like `every 1 minute` will wait 1 minute after each run, which is what we want (overruns would be bad) but requires adjusting your math a bit. It's not quite 5 / minute, more like N per (1m + N * mean_create_bp_latency). Would be OK just saying "approximately" here.

Reply:
Good point – I added a small clarification to the comment.